### PR TITLE
powerflex related updates

### PR DESCRIPTION
**Pull request by gaelrehault**

### CODE REVIEW

File: src/deploy/osp_deployer/deployer.py [hunk 1]
```diff
@@ -233,6 +233,7 @@ def validate_edge_sites_in_settings(args, setts):
 
 
 def deploy_powerflex_gw(setts, sah_node, tester, powerflexgw_vm):
+    settings = setts
```
Comment (jproque-dell) [inline src/deploy/osp_deployer/deployer.py:236]: Is this line needed? we're passing settings as setts, and is called as such afterward.

File: src/deploy/osp_deployer/deployer.py [hunk 2]
```diff
@@ -262,6 +263,7 @@ def deploy_powerflex_gw(setts, sah_node, tester, powerflexgw_vm):
 
 
 def deploy_powerflex_mgmt(setts, sah_node, tester, powerflexmgmt_vm):
+    settings = setts
```
Comment (jproque-dell) [inline src/deploy/osp_deployer/deployer.py:266]: same

**Review by jproque-dell — APPROVED** [review]

File: src/deploy/osp_deployer/deployer.py (hunk 1 above)
Comment (gaelrehault) [reply src/deploy/osp_deployer/deployer.py:236]: see a couple of lines further down : 
settings.powerflexgw_vm.root_password
is called a few times

Comment (jproque-dell) [reply src/deploy/osp_deployer/deployer.py:236]: should be setts.powerflexgw_vm.root_password. I figure I missed to change it.

Comment (gaelrehault) [reply src/deploy/osp_deployer/deployer.py]: done

File: src/deploy/osp_deployer/deployer.py (hunk 2 above)
Comment (gaelrehault) [reply src/deploy/osp_deployer/deployer.py]: done

**Pull request merged**